### ISSUE TITLE
Showing only tabs, *if* tabs exist

### ISSUE THREAD

**Issue by ELLIOTTCABLE** [taohex/lightline-buffer#7]
So, the `buftabline` plugin — which I've used up until recently — shows buffers in the tabline, *unless* you're actually using tabpages. When tabpages exist, it operates as the normal Vim tabline, showing only *tabs*.

At the moment, lightline-buffer does its job admirably well … but makes traditional vim tabpages essentially useless, in those few occasions where they were actually quite useful previously. (It's only once in a blue moon when I *actually* want multiple tab-viewports/windowsets onto my buffers, but when I want them, I *really* want them, you know?)

tl;dr I'd love to see lightline-buffer effectively disable itself (and revert the tabline to its original behaviour) if there is more than one tabpage in the Vim instance! (Or at least an option for that.)

**Comment by polyzen** [comment]
I've also noticed this while switching from airline.

**Comment by taohex** [comment]
Thank you for your advice! I will think about how to optimize it.
